projects/components/src/table/cells/data-parsers/table-cell-object-parser.ts

```diff
@@ -0,0 +1,21 @@
+import { TableCellParser } from '../table-cell-parser';
+import { TableCellParserBase } from '../table-cell-parser-base';
+import { CoreTableCellParserType } from '../types/core-table-cell-parser-type';
+
+@TableCellParser({
+  type: CoreTableCellParserType.Object
+})
+export class TableCellObjectParser extends TableCellParserBase<ObjectCellData, ObjectCellData, string | undefined> {
```

Comment:
Could we make this even more generic? For example, why does it have to even be an object?

The parseValue function could remain as is, and the parseFilterValue function could just have a bit of logic like
```typescript
if (typeof cellData === 'object' && cellData !== null && 'filterValue' in cellData) {
  return cellData.filterValue;
}
return String(cellData);
```

Reply:
Made everything unknown.